_data/projects/MangGym.yml

```diff
@@ -0,0 +1,10 @@
+name: csp1
+desc: My very first Capstone Project about Gym near my house
+site: https://github.com/reviter/csp1
+tags:
+  - html
+  - css
+  - web
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/reviter/csp1/labels/up%20for%20grabs
```

Comment:
There are no issues listed on this repository. Are you interested in creating some and labelling them with `up for grabs` so that contributors can dive in as soon as this is merged and published?

Reply:
Yes sir I'm interested. That's why I put it here on up for grabs so that maybe they could improve my site in a much better way. Thank you sir!

Reply:
Having tasks available means that contributors will know how they can help.

Let me know when those tasks are in place so we can wrap this up!

Reply:
Okay sir. Let me check. I will make some tasks. and i will label them up for grabs right sir? Thank you

Reply:
I created a tasks now sir. I'll add more later. Thank you.